src/main/java/net/openhft/chronicle/bytes/internal/CanonicalPathUtil.java

```diff
@@ -27,7 +28,7 @@
 public final class CanonicalPathUtil {
 
     // This is unlikely to collide with other internalized strings and is seldom the same across JVMs
-    private static final String PREFIX = CanonicalPathUtil.class.getName() + Math.abs(new SecureRandom().nextInt(1 << 30)) + "::";
```

Comment:
I believe that this commit - https://github.com/OpenHFT/Chronicle-Bytes/commit/58d297f9ec11a55378da5a37238ade6c1f7b7647 - broke this behaviour and that we should revert to synchronizing on
```
file.getCanonicalPath().intern()
```

Reply:
See also https://github.com/OpenHFT/Chronicle-Bytes/pull/445